tests/fixtures/ember-container-query-addon/input/addon/components/ui/form/information.d.ts

```diff
@@ -1,4 +1,5 @@
 export interface Args {
+  formId: string;
```

Comment:
**bugfix**

I had copied the fixtures from an earlier version, where `formId` is missing from `Args`.